Pearson-Advance/openedx-lti-tool-plugin
# PADV-656: Add AGS course score update signal

## Ticket

https://agile-jira.pearson.com/browse/PADV-656

## Description

This PR adds a signal that catches course score changes on graded LTI courses. This signal will send a AGS score update to the related graded resource lineitem. It will only send a score update if the signal includes a passed course grade.

## Type of Change

- [x] Import COURSE_GRADE_CHANGED signal from edx-platform.
- [x] Add update_course_score signal receiver.
- [x] Modify LtiGradedResource update_score method.
- [x] Include tests for all added code.

## Testing:

- Run the LMS: `make dev.up.lms`.
- Install `openedx_lti_tool_plugin` on the LMS.
- Run migrations: `./manage.py lms migrate`.
- Run LMS shell: `./manage.py lms shell`.
- Create a course and setup grading.
- Create a LtiGradedResource for a course and LTI user.
- Launch the course used on the created LtiGradedResource.
- Interact with the course until you reach the course pass score.
- A AGS service request call should have been called once a problem submitted reached the course pass grade.

## Reviewers
